docs/setup.md

```diff
@@ -1,28 +1,70 @@
 # Setup
 
-Setting Supibot up locally requires several steps, and is much easier if done with an interactive script.
+### Docker 
+
+The easiest way to get started is to use a docker-compose file, which includes all the needed services:
+```
+version: "3"
 
-### Interactive
+services:
+    db:
+      image: supinic/supidb
+      restart: unless-stopped
+      environment:
+        - MYSQL_RANDOM_ROOT_PASSWORD=1
+        - MYSQL_PASSWORD=supibot
+        - MYSQL_USER=supibot
+    redis:
+      image: redis
+      restart: unless-stopped
+    supibot:
+      links:
+        - db
+        - redis
+      image: supinic/supibot
+      restart: unless-stopped
+      environment:
+        - "MARIA_HOST=db"
+        - "MARIA_USER=supibot"
+        - "MARIA_PASSWORD=supibot"
+        - "REDIS_CONFIGURATION=redis"
+        - "COMMAND_PREFIX=|"
+        - "INITIAL_BOT_NAME=supibot"
+        - "INITIAL_PLATFORM=twitch"
+        - "INITIAL_CHANNEL=supinic"
+        - "TWITCH_CLIENT_ID=<your_client_id>"
+        - "TWITCH_OAUTH=<your_oauth>"
```

Comment:
would it make sense to also include other platforms' oauths here? along with platform being one of the ones we support